elliott/elliottlib/util.py

```
@@ -734,3 +736,255 @@ def get_advisory_boilerplate(runtime, et_data, art_advisory_key, errata_type):
+                        return (image_url, nvr, "")
+                    else:
+                        missing_labels = []
+                        if not component:
+                            missing_labels.append('com.redhat.component')
+                        if not version:
+                            missing_labels.append('version')
+                        if not release:
+                            missing_labels.append('release')
+                        error_msg = f"Missing required labels: {', '.join(missing_labels)}"
+                        logger.debug(f"✗ {error_msg} for {image_url}")
+                        return (image_url, "", error_msg)
```

Pylint (4.0.2)

[refactor] Unnecessary "else" after "return", remove the "else" and de-indent the code inside it

(R1705)